src/components/ImageView/ImageView.js

```diff
@@ -475,10 +475,28 @@ export default observer(
       if (!item.annotation.editable) return;
       if (p && p.className === "Transformer") return;
 
+      // clicking on the stage after there has already been a region selection
+      // should clear selected areas and not continue drawing a new region immediately.
+      if (
+        isFF(FF_DEV_1442) &&
+        e.target === item.stageRef &&
+        item.annotation.selectedRegions.length > 0 &&
+        [
+          undefined,
+          "EllipseTool",
+          "EllipseTool-dynamic",
+          "RectangleTool",
+          "RectangleTool-dynamic",
```

Comment:
We also need this behavior for closed polygons

Reply:
good point, let me add that in!